src/pint/observatory/special_locations.py

```diff
@@ -86,20 +72,19 @@ def __init__(
 class BarycenterObs(SpecialLocation):
     """Observatory-derived class for the solar system barycenter.
 
-    Time scale is assumed to be tdb."""
+    Time scale is assumed to be TDB, so no GPS or BIPM corrections should be applied."""
 
     def __init__(
         self,
         name,
         aliases=None,
+        apply_gps2utc=False,
```

Comment:
Should this even be an option?

Reply:
Is there any scenario where an SSB TOA will have a GPS correction?

Reply:
Good point. Removed.